JavaScript/JavaScriptSDK/snippet.js

```diff
@@ -49,8 +57,8 @@
     createLazyMethod(start + trackEvent);
     createLazyMethod(stop + trackEvent);
 
-    createLazyMethod(start + trackPage);
-    createLazyMethod(stop + trackPage);
+    createLazyMethod(start + trackPage + internal);
```

Comment:
`startTrackPageInternal` is an internal method, there is no need to expose it to users. You need to create a lazy method for the `startTrackPage`. In line 35 you need to modify the originalArguments to change `startTrackPage` to `startTrackPageInternal` (the same for stop).